types/task-worklet/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for task-worklet 0.1
+// Project: https://github.com/GoogleChromeLabs/task-worklet
```

Comment:
Please provide correct project link as clicking on this results in `404`. We need it to see if you are writing correct definitions. Thanks.

Reply:
Done. It must have been extracted by `dts-gen` based on the `repository` field in the related npm package itself.

https://github.com/developit/task-worklet/blob/fa10195446fc757b4f10a4bf5ffa8e35c9b9d049/package.json#L32